docs/product/explore/session-replay/mobile/index.mdx

```diff
@@ -0,0 +1,89 @@
+
+</Note>
+
+Session Replay allows you to see reproductions of user sessions, which can help you understand what happened before, during, and after an error or performance issue occurred. As you play back each session, you'll be able to see every user interaction in relation to network requests, frontend and backend errors, backend spans, and more.
+
+Replays help you see exactly how the user experience is impacted by errors. Because they're integrated with our Issues product, you'll be able to see session replays connected to error events on the [Issue Details](https://docs.sentry.io/product/issues/issue-details/) page in Sentry. To make sure backend errors are also included in the replay, see our [backend set up instructions](https://docs.sentry.io/product/explore/session-replay/getting-started/#replays-for-backend-errors).
+
+## What is Session Replay?
+
+Session Replay for mobile works by taking snapshots of the view hierarchy as well as a screenshot within the same frame, at the rate of one per second. These screenshots are then compressed into a video file representing a segment of the user’s session. All these small segments are then streamed into Sentry with additional trace identifiers, breadcrumbs, and other useful debugging information to compose the full user session.
+
+The user session (replay) lasts until the user closes the app, the app crashes, the user puts the app in the background for over 30 seconds, or the session reaches 60 minutes in duration. The result is a video playback that can help you reproduce bugs in your mobile app, including hard-to-debug issues related to backend APIs and microservices.
+
+To redact anything potentially sensitive and preserve maximum user privacy, the view hierarchy is used to find the position of controls such as text boxes, images, labels, and buttons. Redaction is done with a block that's drawn over these controls, using the most predominant color of the control.
+
+## Replay Details
+
+Every replay has a detailed view that contains the embedded video player and rich debugging context. Playing back the video will allow you to see every user interaction in relation to network requests, frontend and backend errors, backend spans, and more.  Almost every component on this page is connected through timestamps.
+
+The below list shows the breakdown of each component and why it’s valuable:
+
+- **Breadcrumbs:** The replay breadcrumbs show when key user interactions took place, specifically: user taps with the relevant component, navigations, and when the user put the app in background or foreground. Breadcrumbs also provide insight into the device of a given user session at particular timestamps:
+
+- `device battery` (when battery level or charging status changes)
+- `orientation` (when the user rotates the device)
+- `connectivity` (when this status changes between wifi, cellular, and offline).
+
```

Comment:
I think this should be nested 
```suggestion
      - `device battery` (when battery level or charging status changes)
      - `orientation` (when the user rotates the device)
      - `connectivity` (when this status changes between wifi, cellular, and offline).

```

Reply:
Oh I think I see what you mean!

Reply:
Great catch Catherine! thanks
![image](https://github.com/user-attachments/assets/09c1d552-ae72-446b-9c71-0843d7d7f8c9)